docs/admin/Accounts/index.rst

```diff
@@ -0,0 +1,4 @@
+.. admin_Accounts_index:
+
+Accounts
+=========
```

Comment:
One "finicky" thing about RST is that the amount of `=` characters should be the same as the length of the title:

```suggestion
Accounts
========
```

Reply:
@sergei-maertens 
Actually, I found in several RST-manuals that the amount of `=` characters should be **at least as many** as the length of the title. :-)

For instance:
https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html#section-structure 
https://sphinx-tutorial.readthedocs.io/step-1/

Reply:
yes, that's correct, but in our styleguides we use the same amount :)